frontend/src/app/testQueue/TestCardForm/TestCardForm.utils.tsx

```diff
@@ -59,14 +73,29 @@ const filterHIVFromAllDevices = (deviceTypes: QueriedDeviceType[]) => {
   );
 };
 
+const filterSyphilisFromAllDevices = (deviceTypes: QueriedDeviceType[]) => {
+  const filteredDeviceTypes = deviceTypes.map((d) =>
+    filterSyphilisFromDevice(d)
+  );
+
+  return filteredDeviceTypes.filter(
+    (d) => d.supportedDiseaseTestPerformed.length > 0
+  );
+};
+
 export function useFilteredDeviceTypes(facility: QueriedFacility) {
   const hivEnabled = useFeature("hivEnabled");
+  const syphilisEnabled = useFeature("syphilisEnabled");
 
   let deviceTypes = [...facility!.deviceTypes];
 
   if (!hivEnabled) {
     deviceTypes = filterHIVFromAllDevices(deviceTypes);
```

Comment:
Could we refactor `filterHIVFromAllDevices` and `filterSyphilisFromAllDevices` into a single method called `filterDiseaseFromAllDevices` that takes in the deviceTypes as well as one of the values in `MULTIPLEX_DISEASES` enum? We frequently reuse that enum in other parts of the app and it would be good to have the new method strongly typed to use one of those values directly rather than loose strings on lines 47 and 51